### PR TITLE
some views auth/login route changed to /login

### PR DESCRIPTION
**Pull request by tortuvshin**

### CODE REVIEW

**Comment by gocanto** [comment]
Is there anyway you could finish the whole translation? so we can have just one commit for this. 
Thanks fir your help.

**Comment by tortuvshin** [comment]
@gocanto can you take a my pull request? may be later i will return back to again pull request.

File: resources/views/products/detailProd.blade.php [hunk 1]
```diff
@@ -185,7 +185,7 @@
 			                                </div>
 
 		                                @else
-		                                    <a  href="/auth/login btn-sm" class="btn btn-info full-width">
+		                                    <a  href="/login btn-sm" class="btn btn-info full-width">
```
Comment (tortuvshin) [inline resources/views/products/detailProd.blade.php]: @gocanto ok

**Comment by gocanto** [comment]
Yes, I can do it. But please, fix the ```href``` mentioned on the reviews. Thanks for testing!

**Pull request merged**